lib/ui/painting.dart

```diff
@@ -375,6 +398,7 @@ class Paint {
     _data.setInt32(_kIsAntiAliasOffset, encoded, _kFakeHostEndian);
   }
 
+  // Must be kept in sync with the default in paint.cc
```

Comment:
nitty nit nit: You don't have a period here at the end of the comment, but you do in the C++ comment. I'd add a period here.